Sources/QuartzRenderer/QuartzRenderer.swift

```diff
@@ -56,21 +56,13 @@ public class QuartzRenderer: Renderer {
                                  space: Self.colorSpace,
                                  bitmapInfo: Self.bitmapInfo)!
         self.isExternalContext = false
-        
-        let rect = CGRect(origin: .zero, size: CGSize(imageSize))
-        context.setFillColor(Color.white.cgColor)
```

Comment:
Isn't a white background a better default setting? Both SVG and AGG backends have a default whie background.

Reply:
The GraphLayout already fills the area in a background color, so it isn't necessary to do it again.